tools/cxbe/Cdxt.cpp

```diff
@@ -0,0 +1,87 @@
+//FIXME: License
+// (c)2019 Jannik Vogel
+
+#include "Exe.h"
+#include "Common.h"
+
+#include <string.h>
+#include <stdio.h>
+
+// program entry point
+int main(int argc, char *argv[])
+{
+    char szErrorMessage[266] = {0};
+    char szExeFilename[266]  = {0};
+    char szDxtFilename[266]  = {0};
+
+    const char* program = argv[0];
+    const char* program_desc = "CDXT: EXE to DXT Relinker";
+    static Option options[] = {
+        { szExeFilename, NULL,  "exefile"  },
+        { szDxtFilename, "OUT", "filename" },
+        { NULL }
+    };
+
+    if(ParseOptions(argv, argc, options, szErrorMessage)) {
+        goto cleanup;
+    }
+
+    // verify we recieved the required parameters
+    if(szExeFilename[0] == '\0')
+    {
+        ShowUsage(program, program_desc, options);
+        return 1;
+    }
+
+    // if we don't have an Dxt filename, generate one from szExeFilename
+    if(szDxtFilename[0] == '\0')
+    {
+        GenerateFilename(szDxtFilename, ".dxt", szExeFilename, ".EXE");
+    }
+
+    // open and convert Exe file
+    {
+        Exe *ExeFile = new Exe(szExeFilename);
+
+        if(ExeFile->GetError() != 0)
+        {
+            strcpy(szErrorMessage, ExeFile->GetError());
+            goto cleanup;
+        }
+
+        //FIXME: Fix subsystem and other header fields?
+
```

Comment:
Force existence of import table.
Currently forced by explicitly importing something in user-code.
Will probably forced by auto-linked dummy in the future.
Could also be forced here (for users who don't link with nxdk).